test/run_tests.sh

```diff
@@ -47,27 +47,11 @@ fi
 # fi
 echo "Skipping var_action_test/run_test until feature is fully implemented"
 
-# Run doctests
-./run_doctest.sh
-res=$?
-errcnt=$((errcnt + res))
-if [ $res -ne 0 ]; then
-  echo "${errcnt} doctest failures"
-fi
-
-for test in `ls unit_tests/test_*.py`; do
-  echo "Running unit test, ${test}"
-  python3 ${test}
-  res=$?
-  errcnt=$((errcnt + res))
-  if [ $res -ne 0 ]; then
-    echo "Failure, '${res}', running unit test, ${test}"
-  fi
-done
+pytest -v unit_tests/
```

Comment:
I think we need to either bring back `run_doctest.sh` or add a `pytest --doctest-modules` line if we expect this shell script to locally run all tests, or instead remove `pytest` as we want this script to only test the generated Fortran code.  I think I am happy with either option (and pinging @peverwhee in case she has an opinion).

Also, it might be good to add a flag to this script to skip the python testing completely if we want to (and we end up keeping it), and then use that flag in the `capgen_unit_tests` workflow to make sure that Github Action is only performing the Fortran unit-testing.

Reply:
@nusbaume I do not have an opinion either way and leave it up to @mwaxmonsky on how he wants to implement #helpful

Reply:
I'm all for less is more so I think removing the `pytest` call is reasonable and keeps the fortran testing separate from the pure python.